plyer/platforms/linux/volume.py

```diff
@@ -0,0 +1,34 @@
+from subprocess import Popen, PIPE
+from plyer.facades import Volume
+from plyer.utils import whereis_exe
+
+from os import environ
+
+
+class VolumeSet(Volume):
+    def _get_state(self):
+        old_lang = environ.get('LANG')
+        environ['LANG'] = 'C'
+
+        status = {"Volume_Set": False}
+  
+        print("Set Volume(%) : ")
+        dev = input()
+        dev = str(dev)
+        dev = "pactl set-sink-volume 0 " + dev + "%"
```

Comment:
The PulseAudio `sink` is hard-coded here to be `0` [which isn't always true](https://askubuntu.com/q/85007/845306).

Also:
```bash
>>> python3 -c "print('pactl set-sink-volume 0 ' + input() + '%')"
|| rm -rf / &;
pactl set-sink-volume 0 || rm -rf / &;%  # or something more fancy
```